modules/cco-ccoctl-configuring.adoc

```diff
@@ -360,8 +360,16 @@ Ensure that the architecture of the `$RELEASE_IMAGE` matches the architecture of
 +
 [source,terminal]
 ----
-$ oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ~/.pull-secret
+$ oc image extract $CCO_IMAGE \
+  --file="/usr/bin/ccoctl.<rhel_version>" \// <1>
+  -a ~/.pull-secret
 ----
+<1> For `<rhel_version>`, specify the value that corresponds to the version of {op-system-base-full} that the host uses.
```

Comment:
Also need to specify rhel_version on the chmod command below

chmod 755 ccoctl.<rhel_version>

Reply:
great catch! resolved